draft-ietf-quic-ack-frequency.md

```diff
@@ -365,6 +365,17 @@ reducing the ACK rate compared to {{Section 13.2.1 of QUIC-TRANSPORT}} during
 extreme congestion or when peers are using DCTCP {{?RFC8257}} or other
 congestion controllers that mark more frequently than classic ECN {{?RFC3168}}.
 
+If the most recent ACK_FREQUENCY frame an endpoint has received from the peer
+has an `Ignore CE` value of `true` (0x01), receipt of a CE marked packet
+SHOULD NOT cause an endpoint to send an immediate acknowledgement.  The endpoint
+still sends an immediate acknowledgement if it would have for a non CE marked
+packet.  If an immediate acknowledgement is not sent, the CE marks are reported
+in the next acknowledgement.
+
+The Ignore-CE bit SHOULD NOT be set if the sender sets ECT(1) in its outgoing
+packets, such as with L4S, because it delays the congestion controller's ability
+to quickly respond to congestion.
+
```

Comment:
This concerns me: If a transport has decided to set ECT ... be it ECT(0) or ECT(1), I would expect it is obliged to respond to CE marks it receives. In what conditions does it become OK for that transport to then say it will not react to CE-marking, and decide it will ignore it?

Reply:
This issue was discussed in detail here - https://github.com/quicwg/ack-frequency/issues/87

I don't think it is ok for the sender to respond to CE marking at all, but it could be that the receiver could send ACKs on transition from ECT(0) to CE or CE to ECT(0) to reduce the number of ACKs. I am not fully convinced with Ignore_CE being sent from the sender as I don't think it conveys the right intention. A better way would be for the receiver to decide when to send ACKs when CE or ECT(0 or 1) are received.

Reply:
@gorryfair -- this isn't saying that the receiver should not respond to CE marks, it is reducing the urgency with which the receiver communicates this information to the sender... I think you might have an issue with having this field in the first place. I would take that up as a separate issue, but I'm happy to weigh in there. I'm personally not strongly attached to this bit.

I'll merge this PR anyway, since it isn't the thing that there's disagreement on here.